File: src/main/java/com/chiu/Hello.java
package com.chiu;

public class Hello {
    public static void main(String[] args) {
        //System.out.println("Hello World!!");
        //new Person().Hello();
        Person p = new Person();
        p.Hello();

        char a='A';// 只能儲存１個字元
        Character b = 'c';
        //float weight = 55.2; 在Java小數點一律使用double
        float weight1 = 55.2f;//加上f就可以放在float裡了
        Float x = Float.valueOf(25);
        Double weight = 55.2;
        String name = "邱俊鈐";
        Integer age2 = 19;

        System.out.println(b);


    }

}

File: src/main/kotlin/com/chiu/Hello.kt
package com.chiu

fun main(args: Array<String>) {
    //Human().Hello()
    val h=Human()
    val age =19 //也可以寫成 val age : int =19
    h.Hello()


}

//建立類別
class Human{
   //建立方法
    fun Hello(){
       print("Good!!")
   }
}